pkg/proxy/ipvs/proxier.go

```diff
@@ -453,6 +454,61 @@ func NewProxier(ipt utiliptables.Interface,
 	return proxier, nil
 }
 
+func NewDualStackProxier(
+	ipt utiliptables.Interface,
+	ipvs utilipvs.Interface,
+	ipset utilipset.Interface,
+	sysctl utilsysctl.Interface,
+	exec utilexec.Interface,
+	syncPeriod time.Duration,
+	minSyncPeriod time.Duration,
+	excludeCIDRs []string,
+	strictARP bool,
+	masqueradeAll bool,
+	masqueradeBit int,
+	clusterCIDR string,
+	hostname string,
+	nodeIP net.IP,
+	recorder record.EventRecorder,
+	healthzServer healthcheck.HealthzUpdater,
+	scheduler string,
+	nodePortAddresses []string,
+) (*MetaProxier, error) {
+
+	// Create an ipv4 instance of the single-stack proxier
+	ipv4Proxier, err := NewProxier(ipt, ipvs, ipset, sysctl,
+		exec, syncPeriod, minSyncPeriod, excludeCIDRs, strictARP,
+		masqueradeAll, masqueradeBit, clusterCIDR, hostname, nodeIP,
+		recorder, healthzServer, scheduler, nodePortAddresses)
+	if err != nil {
+		return nil, fmt.Errorf("unable to create ipv4 proxier: %v", err)
+	}
+
+	// Create an ipv6 instance of the single-stack proxier
+	
+	// TODO: Remove hard-coded values
```

Comment:
When can we remove these?

Reply:
My problem is that the necessary config updates is in PR https://github.com/kubernetes/kubernetes/pull/79386. Please see https://github.com/kubernetes/kubernetes/pull/80627#discussion_r316083728.

I will remove the hard-coded values from `ipvs/proxier.go` and use the dual-stack parameters, with slice/map for parameters `iptablesInterfaces`, `clusterCIDRs` and `nodeIPs`;

https://github.com/vllry/kubernetes/blob/a33a33f28da0900262810a700d4efb836ce63bba/cmd/kube-proxy/app/server_others.go#L216